ntoskrnl/cc/view.c

```diff
@@ -595,34 +595,26 @@ CcRosUnmarkDirtyVacb (
     }
 }
 
-static
 BOOLEAN
-CcRosFreeUnusedVacb (
-    PULONG Count)
+CcRosFreeOneUnusedVacb (
+    void)
```

Comment:
```suggestion
CcRosFreeOneUnusedVacb(VOID)
```

Reply:
I'l uppercase the void but I'll leave the indentation, as that follows the surrounding code.